integration_tests/src/main/python/date_time_test.py

```diff
@@ -13,14 +13,17 @@
 # limitations under the License.
 
 import pytest
-from asserts import assert_gpu_and_cpu_are_equal_collect, assert_gpu_fallback_collect
+from asserts import assert_gpu_and_cpu_are_equal_collect, assert_gpu_fallback_collect, assert_gpu_and_cpu_error
 from data_gen import *
 from datetime import date, datetime, timezone
 from marks import incompat, allow_non_gpu
 from pyspark.sql.types import *
 from spark_session import with_spark_session, is_before_spark_330
 import pyspark.sql.functions as f
 
+ANSI_ENABLE_CONF = {'spark.sql.ansi.enabled': 'true'}
+LEGACY_INTERVAL_CONF = {'spark.sql.legacy.interval.enabled': 'true'}
```

Comment:
Nit: This should be moved next to the other predefined confs in this file

Reply:
Done.